Documentation/running-exporters.md

```diff
@@ -0,0 +1,66 @@
+# Running Exporters
+Running exporters and scraping them with prometheus-operator.
```

Comment:
As the Prometheus Operator doesn't actually scrape anything this sentence should be rewritten to something like: "Running exporters and configuring Prometheus to scrape them using means provided by the Prometheus Operator."